### PR TITLE
[fix bug 1224319] Win10 /welcome testing round 3

### PR DESCRIPTION
**Pull request by jpetto**
- Sets winner of previous test as default

### CODE REVIEW

File: media/css/firefox/win10-welcome-variants.less [hunk 1]
```diff
@@ -1,33 +1,50 @@
...
-        display: none;
+@media only screen and (max-width: @breakDesktop) {
+    #win10-welcome-v3-10 .firefox-default-cta {
+        padding-left: 417px;
+        min-height: 251px;
+
+        &:before {
+            width: 377px;
+            height: 251px;
+            background-size: 377px 251px;
```
Comment (schalkneethling) [inline media/css/firefox/win10-welcome-variants.less]: `.background-size();` mixin here?

File: media/css/firefox/win10-welcome-variants.less [hunk 2]
```diff
@@ -1,33 +1,50 @@
...
-        max-width: 12em;
-        margin: 0 auto 12px;
+@media only screen and (max-width: @breakMobileLandscape) {
+    #win10-welcome-v3-10 .firefox-default-cta {
+        padding-top: 230px;
+
+        &:before {
+            width: 300px;
+            height: 210px;
+            background-size: 300px 210px;
```
Comment (schalkneethling) [inline media/css/firefox/win10-welcome-variants.less]: use mixin?

File: media/css/firefox/win10-welcome-variants.less [hunk 3]
```diff
@@ -1,33 +1,50 @@
...
+    padding-left: 490px;
+    min-height: 315px;
 
-#win10-welcome-v2-4 h1.not-default {
-    display: none;
+    &:before {
+        width: 450px;
+        height: 315px;
+        background-image: url('/media/img/firefox/whatsnew_40/win10.animated.gif');
+        background-size: 450px 315px;
```
Comment (schalkneethling) [inline media/css/firefox/win10-welcome-variants.less]: Also consider using the mixin here.

**Comment by schalkneethling** [comment]
Not sure if I am missing something locally but, when I load:
http://localhost:8000/en-US/firefox/windows-10/welcome/

I get:

screen shot 2015-12-15 at 11 44 36

Whilst loading the same URL on demo3, I get

screen shot 2015-12-15 at 11 45 42

**Comment by schalkneethling** [comment]
I am seeing the same behavior for all of the variations. Also tested in Fx on Win10 with the same result.

**Comment by jpetto** [comment]
Ah yes, you need to have UITour enabled for demo3 in your browser to get the proper content. Should have specified when I opened the PR.

**Comment by jpetto** [comment]
Updated variants LESS to use `.background-size` mixin.

**Comment by schalkneethling** [comment]
r+ :ramen:

**Pull request merged**